src/App.jsx

```diff
@@ -10,8 +10,7 @@ function App() {
   return (
     //Any Routes added here won't be merged, please check the routes.jsx file, follow the pattern and add any additional routes.
     <BrowserRouter>
-      <Routes>
-      </Routes>
+      <Routes></Routes>
```

Comment:
Kindly pull from the development branch to be in sync with the project. This line should be 
`<Routes />`

Since we are already importing the routes function from the Routes.jsx file